home.css

```diff
@@ -0,0 +1,137 @@
+@import url("./main.css");
+@import url("./utility.css");
+
+body::-webkit-scrollbar {
+  width: 0.25rem;
+}
+
+body::-webkit-scrollbar-track {
+  background: var(--secondary-white);
+}
+
+body::-webkit-scrollbar-thumb {
+  background: var(--primary-color);
+}
+
+header {
+  box-sizing: border-box;
+  background-color: var(--primary-color);
+  display: flex;
+  align-items: center;
+  /* justify-content: space-between; */
+  padding: 2rem;
+  box-shadow: 0 3px 5px -1px #0003, 0 6px 10px #00000024, 0 1px 18px #0000001f;
+}
+
+header > h3 {
+  margin: 0 20px;
+}
+.hamburger-menu {
+  display: relative;
```

Comment:
this property value is not valid. You might be looking for the `position` property

Reply:
resolved